prediction/tournament_output.py

```diff
@@ -0,0 +1,22 @@
+class TournamentOutput(object):
+    """An abstract data that represents the Tournament output of some model."""
+    def get_winner_prob(self, outcome):
+        pass
+
+    def get_nonzero_prob_outcomes_list(self):
+        pass
+
+class ConcreteTournamentOutput(TournamentOutput):
+
+    def __init__(self, tournament):
+        self.tournament = tournament
+    
+    def get_winner_prob(self,outcome):
```

Comment:
this makes no sense? What is an "outcome" and a "winner" in the "get_winner_prob" function?